packages/wonder-blocks-data/package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "@khanacademy/wonder-blocks-data",
+  "version": "1.0.0",
```

Comment:
Maybe starting with a previous version would be better (e.g. `0.0.1`). If I'm not wrong, lerna will automatically bump this version.

Reply:
Sure, I can do that.